runtime/doc/message.txt

```diff
@@ -315,10 +315,9 @@ generate this message.  This could be caused by a |-m| commandline argument.
 You can switch the 'write' option on with ":set write".
 
 							*E25*  >
-  GUI cannot be used: Not enabled at compile time
+  Nvim does not have a built-in GUI
```

Comment:
@Pyrohh It did take a while. Though it's amazing what the `grep -ri` sledge hammer can do for you :)